### PR TITLE
3.4 master sync

### PR DESCRIPTION
**Pull request by hmaarrfk**
Checklist
* [ ] Used a fork of the feedstock to propose changes
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -136,7 +140,8 @@ test:
         ] %}
         - export MACOSX_DEPLOYMENT_TARGET={{ MACOSX_DEPLOYMENT_TARGET }}       # [osx]
         - export CONDA_BUILD_SYSROOT={{ CONDA_BUILD_SYSROOT }}                 # [osx]
-        - $CXX $RECIPE_DIR/test.cpp -I$PREFIX/include  -L$PREFIX/lib -o test   # [unix]
+        - OPENCV_FLAGS=`pkg-config --cflags opencv4`                           # [unix]
```
Comment (hajapy) [inline recipe/meta.yaml]: should this say opencv3?

File: recipe/meta.yaml [hunk 2]
```diff
@@ -70,8 +66,6 @@ requirements:
     - libtiff
     - libpng
     - openblas                       # [not win]
-    - blas 1.1 {{ blas_variant }}    # [not win]
-    - ffmpeg 4.1.*                   # [not win]
```
Comment (hajapy) [inline recipe/meta.yaml]: master recipe still has ffmpeg 4.1.* in run section

Comment (hmaarrfk) [reply recipe/meta.yaml]: oops, wrong line deleted.

File: recipe/meta.yaml [hunk 3]
```diff
@@ -23,8 +22,6 @@ build:
   number: 1
```
Comment (hajapy) [inline recipe/meta.yaml]: should bump build num

Comment (hmaarrfk) [reply recipe/meta.yaml]: Thanks for the thorough review.

Comment (hmaarrfk) [reply recipe/meta.yaml]: Let's do that after we find that the tests pass or fail

File: recipe/meta.yaml (hunk 1 above)
Comment (hmaarrfk) [reply recipe/meta.yaml]: maybe. lets see

File: recipe/meta.yaml [hunk 4]
```diff
@@ -136,7 +140,8 @@ test:
         ] %}
         - export MACOSX_DEPLOYMENT_TARGET={{ MACOSX_DEPLOYMENT_TARGET }}       # [osx]
         - export CONDA_BUILD_SYSROOT={{ CONDA_BUILD_SYSROOT }}                 # [osx]
-        - $CXX $RECIPE_DIR/test.cpp -I$PREFIX/include  -L$PREFIX/lib -o test   # [unix]
+        - OPENCV_FLAGS=`pkg-config --cflags opencv2`                           # [unix]
```
Comment (hajapy) [inline recipe/meta.yaml]: Maybe just `opencv`

Comment (hmaarrfk) [reply recipe/meta.yaml]: success.

**Comment by hajapy** [comment]
Thanks for this!

**Comment by hmaarrfk** [comment]
now to build with qt5.12

**Pull request merged**